docs/source/settings.rst

```diff
@@ -168,3 +168,14 @@ COMMENT_DEFAULT_PROFILE_PIC_LOC
 
 Provides an alternate location for profile picture that can be used other than
 default image. Defaults to '/static/img/default.png'
+
+
+COMMENT_ALLOW_BLOCKING_USERS
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Enable blocking system. This gives only **admins** the right.
+
+COMMENT_ALLOW_MODERATOR_TO_BLOCK
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Allow **moderators** to perform blocking action when `COMMENT_ALLOW_BLOCKING_USERS`_ is enabled.
```

Comment:
It would be informative to add the default values here. The same is done for other available configurations.